sdks/java/io/amazon-web-services/src/test/java/org/apache/beam/sdk/io/aws/s3/S3WritableByteChannelTest.java

```diff
@@ -148,9 +148,9 @@ public void testAtMostOne() {
     assertTrue(atMostOne(false, false));
     assertFalse(atMostOne(true, true, true));
     assertFalse(atMostOne(true, true, false));
-    assertFalse(atMostOne(true, false, true));
```

Comment:
This test intentionally includes extra boolean values, so I don't think it should be changed.